pallets/bridge/src/mock.rs

```diff
@@ -221,35 +212,35 @@ impl FindAuthor<u64> for AuthorGiven {
 
 // Implement FRAME authorship pallet configuration trait for the mock runtime
 impl pallet_authorship::Config for MockRuntime {
+	type EventHandler = ();
+	type FilterUncle = ();
 	type FindAuthor = AuthorGiven;
 	type UncleGenerations = ();
-	type FilterUncle = ();
-	type EventHandler = ();
 }
 
 // Implement FRAME timestamp pallet configuration trait for the mock runtime
 impl pallet_timestamp::Config for MockRuntime {
+	type MinimumPeriod = ();
 	type Moment = u64;
 	type OnTimestampSet = ();
-	type MinimumPeriod = ();
 	type WeightInfo = ();
 }
 
 // Parameterize Centrifuge Chain chainbridge pallet
 parameter_types! {
 	pub const MockChainId: u8 = TEST_CHAIN_ID;
-	pub const ChainBridgePalletId: PalletId = common_types::ids::CHAIN_BRIDGE_PALLET_ID;
+	pub const ChainBridgePalletId: PalletId = cfg_types::ids::CHAIN_BRIDGE_PALLET_ID;
```

Comment:
Would be good to have a unified pattern how we use these imports. Sometimes we use the whole path, sometimes we `use` the imported type from the top.

Reply:
Agree, I like whole paths for the ids. But won't change it in this PR.